<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>923</width>
    <height>695</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout"/>
    </item>
    <item>
     <widget class="QCheckBox" name="m_PauseVoipCheckBox">
      <property name="text">
       <string>Pause VOIP (VOIP is simulated with 400Hz tone Mono Channel PCM)</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4"/>
    </item>
    <item>
     <widget class="QCheckBox" name="m_PauseKodiCheckbox">
      <property name="text">
       <string>Pause Kodi (Kodi is simulated with 100Hz tone Stereo Channel Float)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="m_PauseMicrophoneCheckBox">
      <property name="text">
       <string>Pause MicrophoneInput</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="m_MuteSpeakersCheckBox">
      <property name="text">
       <string>Mute Speakers</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Speaker Volume</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="m_SpeakerVolumeSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Speaker Output Peak Level</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="m_AudioOutPeakProgressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="m_MuteMicrophoneCheckBox">
      <property name="text">
       <string>Mute Microphone</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Microphone Volume</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="m_MicrophoneVolumeSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Microphone Input Peak Level</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="m_AudioInPeakProgressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Waveform" name="m_WaveForm" native="true"/>
    </item>
    <item>
     <widget class="LogWidget" name="m_LogWidget"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Waveform</class>
   <extends>QWidget</extends>
   <header location="global">QtSoundTest/WaveForm.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LogWidget</class>
   <extends>QPlainTextEdit</extends>
   <header location="global">LogWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
